scipy/optimize/_minpack_py.py

```diff
@@ -590,14 +591,19 @@ def curve_fit(f, xdata, ydata, p0=None, sigma=None, absolute_sigma=False,
         and raise a ValueError if they do. Setting this parameter to
         False may silently produce nonsensical results if the input arrays
         do contain nans. Default is True.
-    bounds : 2-tuple of array_like, optional
+    bounds : 2-tuple of array_like or `Bounds`, optional
         Lower and upper bounds on parameters. Defaults to no bounds.
-        Each element of the tuple must be either an array with the length equal
-        to the number of parameters, or a scalar (in which case the bound is
-        taken to be the same for all parameters). Use ``np.inf`` with an
-        appropriate sign to disable bounds on all or some parameters.
+        There are two ways to specify the bounds:
 
-        .. versionadded:: 0.17
+            - Instance of `Bounds` class.
+
+            - 2-tuple of array_like: Each element of the tuple must be either
+              an array with the length equal to the number of parameters, or a
+              scalar (in which case the bound is taken to be the same for all
+              parameters). Use ``np.inf`` with an appropriate sign to disable
+              bounds on all or some parameters.
+
+              .. versionadded:: 0.17
```

Comment:
At that point I would remove this or we should add a `versionchanged` to the other bullet if we want to be extra correct.

Reply:
I also think the `curve_fit` docs are overloaded with "New in version x" boxes. But will let maintainers decide, seems like a policy issue.